spec/controllers/graphql/queries/freak_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module Graphql
+  RSpec.describe GraphqlController, type: :controller do
+    subject(:query_freaks) { post :execute, params: params, as: :json }
+
+    let(:params) do
+      {
+        query: File.read('spec/fixtures/requests/queries/freak.graphql')
+      }
+    end
+
+    before do
+      create(:freak, id: 1, first_name: 'Ion', description: 'the best')
```

Comment:
Is it necessary to specify the `id` here? 

From experience, it's a bad practice to specify id's when creating entities.